srcpkgs/inspircd/template

```diff
@@ -30,7 +30,6 @@ do_configure() {
 	./configure \
 		--enable-extras "geo_maxmind regex_posix regex_stdlib ssl_gnutls ssl_openssl sslrehashsignal sqlite3"
 	./configure \
-		--binary-dir /usr/bin \
```

Comment:
Since 3.13.0 the default binary dir changed from /usr/sbin to /usr/bin so this is no longer necessary.